lib/platform/git/storage.ts

```diff
@@ -259,7 +274,12 @@ export class Storage {
     if (!files) {
       return [];
     }
-    return files.split('\n').filter(Boolean);
+    return files
```

Comment:
Is this cached, or recalculated every time it's called?

Reply:
No caching, it's recalculated every time. I know it's O(n^2), but it's rarely called, and seems to be okay with a large number of submodules (I tested with RedhawkSDR/redhawk which has 53 files, 49 of which are submodules).